final_task/rss_reader/add_to_csv.py

```diff
@@ -0,0 +1,17 @@
+import csv
+
+def addcsv(articles):
+    with open('news.csv', 'a', newline='') as csvfile:
+        fieldnames = ['link','title', 'img', 'summary', 'published']
+
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+
+        writer.writerow(articles)
+
+def out():
+    a = []
+    with open('news.csv', 'r') as csvFile:
+        reader = csv.reader(csvFile)
+        for row in reader:
+            a.append(row)
```

Comment:
If there is no such file (and the first time you launch program on new machine there won't be such file) this line will crash your application with `FileNotFoundError`